system_metrics_collector/src/system_metrics_collector/linux_memory_measurement_node.cpp

```diff
@@ -124,4 +125,30 @@ double LinuxMemoryMeasurementNode::periodicMeasurement()
   return processMemInfoLines(read_string);
 }
 
+void LinuxMemoryMeasurementNode::publishStatistics()
+{
+  if (publisher_ == nullptr) {
+    return;
+  }
+
+  const moving_average_statistics::StatisticData statistic_data = getStatisticsResults();
+  const double data[moving_average_statistics::STATISTICS_DATA_TYPES.size()] = {
```

Comment:
`std::array`

Reply:
This kind of code is no longer part of this PR.